build errors on Fedora s390x

<details>

```
[300/329] /usr/bin/g++  -I/builddir/build/BUILD/blend2d-5a263ce51f3f880ee6c60f6345d18c3eccbe200f/src -O2 -flto=auto -ffat-lto-objects -fexceptions -g -grecord-gcc-switches -pipe -Wall -Werror=format-security -Wp,-U_FORTIFY_SOURCE,-D_FORTIFY_SOURCE=3 -Wp,-D_GLIBCXX_ASSERTIONS -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -fstack-protector-strong -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1  -m64 -march=z13 -mtune=z14 -fasynchronous-unwind-tables -fstack-clash-protection -O2 -g -DNDEBUG -fvisibility=hidden -Wall -Wextra -fno-exceptions -fno-rtti -fno-math-errno -fno-trapping-math -fno-finite-math-only -fno-threadsafe-statics -fno-semantic-interposition -DBL_BUILD_NO_JIT -DBL_TEST -DBL_STATIC -O2 -fno-enforce-eh-specs -fmerge-all-constants -ftree-vectorize -MD -MT CMakeFiles/bl_test_unit.dir/src/blend2d/unicode/unicode_test.cpp.o -MF CMakeFiles/bl_test_unit.dir/src/blend2d/unicode/unicode_test.cpp.o.d -o CMakeFiles/bl_test_unit.dir/src/blend2d/unicode/unicode_test.cpp.o -c /builddir/build/BUILD/blend2d-5a263ce51f3f880ee6c60f6345d18c3eccbe200f/src/blend2d/unicode/unicode_test.cpp
FAILED: CMakeFiles/bl_test_unit.dir/src/blend2d/unicode/unicode_test.cpp.o 
/usr/bin/g++  -I/builddir/build/BUILD/blend2d-5a263ce51f3f880ee6c60f6345d18c3eccbe200f/src -O2 -flto=auto -ffat-lto-objects -fexceptions -g -grecord-gcc-switches -pipe -Wall -Werror=format-security -Wp,-U_FORTIFY_SOURCE,-D_FORTIFY_SOURCE=3 -Wp,-D_GLIBCXX_ASSERTIONS -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -fstack-protector-strong -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1  -m64 -march=z13 -mtune=z14 -fasynchronous-unwind-tables -fstack-clash-protection -O2 -g -DNDEBUG -fvisibility=hidden -Wall -Wextra -fno-exceptions -fno-rtti -fno-math-errno -fno-trapping-math -fno-finite-math-only -fno-threadsafe-statics -fno-semantic-interposition -DBL_BUILD_NO_JIT -DBL_TEST -DBL_STATIC -O2 -fno-enforce-eh-specs -fmerge-all-constants -ftree-vectorize -MD -MT CMakeFiles/bl_test_unit.dir/src/blend2d/unicode/unicode_test.cpp.o -MF CMakeFiles/bl_test_unit.dir/src/blend2d/unicode/unicode_test.cpp.o.d -o CMakeFiles/bl_test_unit.dir/src/blend2d/unicode/unicode_test.cpp.o -c /builddir/build/BUILD/blend2d-5a263ce51f3f880ee6c60f6345d18c3eccbe200f/src/blend2d/unicode/unicode_test.cpp
/builddir/build/BUILD/blend2d-5a263ce51f3f880ee6c60f6345d18c3eccbe200f/src/blend2d/unicode/unicode_test.cpp: In function ‘void bl::Tests::unit_unicode_entry()’:
/builddir/build/BUILD/blend2d-5a263ce51f3f880ee6c60f6345d18c3eccbe200f/src/blend2d/unicode/unicode_test.cpp:31:59: error: expected ‘}’ before ‘{’ token
   31 |     #define ENTRY(DST, DST_ENC, SRC, SRC_ENC, ERROR_CODE) { \
      |                                                           ^
/builddir/build/BUILD/blend2d-5a263ce51f3f880ee6c60f6345d18c3eccbe200f/src/blend2d/unicode/unicode_test.cpp:91:5: note: in expansion of macro ‘ENTRY’
   91 |     ENTRY("\0\0\0T\0\0\0e\0\0\0s\0\0\0t"    , UTF32 , "Test"                            , LATIN1, BL_SUCCESS),
      |     ^~~~~
/builddir/build/BUILD/blend2d-5a263ce51f3f880ee6c60f6345d18c3eccbe200f/src/blend2d/unicode/unicode_test.cpp:30:42: note: to match this ‘{’
   30 |   static const TestEntry testEntries[] = {
      |                                          ^
/builddir/build/BUILD/blend2d-5a263ce51f3f880ee6c60f6345d18c3eccbe200f/src/blend2d/unicode/unicode_test.cpp:31:59: error: expected ‘,’ or ‘;’ before ‘{’ token
   31 |     #define ENTRY(DST, DST_ENC, SRC, SRC_ENC, ERROR_CODE) { \
      |                                                           ^
/builddir/build/BUILD/blend2d-5a263ce51f3f880ee6c60f6345d18c3eccbe200f/src/blend2d/unicode/unicode_test.cpp:91:5: note: in expansion of macro ‘ENTRY’
   91 |     ENTRY("\0\0\0T\0\0\0e\0\0\0s\0\0\0t"    , UTF32 , "Test"                            , LATIN1, BL_SUCCESS),
      |     ^~~~~
/builddir/build/BUILD/blend2d-5a263ce51f3f880ee6c60f6345d18c3eccbe200f/src/blend2d/unicode/unicode_test.cpp:91:110: error: expected primary-expression before ‘,’ token
   91 |     ENTRY("\0\0\0T\0\0\0e\0\0\0s\0\0\0t"    , UTF32 , "Test"                            , LATIN1, BL_SUCCESS),
      |                                                                                                              ^
/builddir/build/BUILD/blend2d-5a263ce51f3f880ee6c60f6345d18c3eccbe200f/src/blend2d/unicode/unicode_test.cpp:31:59: error: expected primary-expression before ‘{’ token
   31 |     #define ENTRY(DST, DST_ENC, SRC, SRC_ENC, ERROR_CODE) { \
      |                                                           ^
/builddir/build/BUILD/blend2d-5a263ce51f3f880ee6c60f6345d18c3eccbe200f/src/blend2d/unicode/unicode_test.cpp:92:5: note: in expansion of macro ‘ENTRY’
   92 |     ENTRY("\0\0\0T\0\0\0e\0\0\0s\0\0\0t"    , UTF32 , "Test"                            , UTF8  , BL_SUCCESS),
      |     ^~~~~
/builddir/build/BUILD/blend2d-5a263ce51f3f880ee6c60f6345d18c3eccbe200f/src/blend2d/unicode/unicode_test.cpp:92:110: error: expected primary-expression before ‘,’ token
   92 |     ENTRY("\0\0\0T\0\0\0e\0\0\0s\0\0\0t"    , UTF32 , "Test"                            , UTF8  , BL_SUCCESS),
      |                                                                                                              ^
/builddir/build/BUILD/blend2d-5a263ce51f3f880ee6c60f6345d18c3eccbe200f/src/blend2d/unicode/unicode_test.cpp:31:59: error: expected primary-expression before ‘{’ token
   31 |     #define ENTRY(DST, DST_ENC, SRC, SRC_ENC, ERROR_CODE) { \
      |                                                           ^
/builddir/build/BUILD/blend2d-5a263ce51f3f880ee6c60f6345d18c3eccbe200f/src/blend2d/unicode/unicode_test.cpp:96:5: note: in expansion of macro ‘ENTRY’
   96 |     ENTRY(""                                , UTF8  , "\xC5"                            , UTF8  , BL_ERROR_DATA_TRUNCATED),
      |     ^~~~~
/builddir/build/BUILD/blend2d-5a263ce51f3f880ee6c60f6345d18c3eccbe200f/src/blend2d/unicode/unicode_test.cpp:96:123: error: expected primary-expression before ‘,’ token
   96 |     ENTRY(""                                , UTF8  , "\xC5"                            , UTF8  , BL_ERROR_DATA_TRUNCATED),
      |                                                                                                                           ^
/builddir/build/BUILD/blend2d-5a263ce51f3f880ee6c60f6345d18c3eccbe200f/src/blend2d/unicode/unicode_test.cpp:31:59: error: expected primary-expression before ‘{’ token
   31 |     #define ENTRY(DST, DST_ENC, SRC, SRC_ENC, ERROR_CODE) { \
      |                                                           ^
/builddir/build/BUILD/blend2d-5a263ce51f3f880ee6c60f6345d18c3eccbe200f/src/blend2d/unicode/unicode_test.cpp:97:5: note: in expansion of macro ‘ENTRY’
   97 |     ENTRY(""                                , UTF8  , "\xEF"                            , UTF8  , BL_ERROR_DATA_TRUNCATED),
      |     ^~~~~
/builddir/build/BUILD/blend2d-5a263ce51f3f880ee6c60f6345d18c3eccbe200f/src/blend2d/unicode/unicode_test.cpp:97:123: error: expected primary-expression before ‘,’ token
   97 |     ENTRY(""                                , UTF8  , "\xEF"                            , UTF8  , BL_ERROR_DATA_TRUNCATED),
      |                                                                                                                           ^
/builddir/build/BUILD/blend2d-5a263ce51f3f880ee6c60f6345d18c3eccbe200f/src/blend2d/unicode/unicode_test.cpp:31:59: error: expected primary-expression before ‘{’ token
   31 |     #define ENTRY(DST, DST_ENC, SRC, SRC_ENC, ERROR_CODE) { \
      |                                                           ^
/builddir/build/BUILD/blend2d-5a263ce51f3f880ee6c60f6345d18c3eccbe200f/src/blend2d/unicode/unicode_test.cpp:98:5: note: in expansion of macro ‘ENTRY’
   98 |     ENTRY(""                                , UTF8  , "\xEF\xBC"                        , UTF8  , BL_ERROR_DATA_TRUNCATED),
      |     ^~~~~
/builddir/build/BUILD/blend2d-5a263ce51f3f880ee6c60f6345d18c3eccbe200f/src/blend2d/unicode/unicode_test.cpp:98:123: error: expected primary-expression before ‘,’ token
   98 |     ENTRY(""                                , UTF8  , "\xEF\xBC"                        , UTF8  , BL_ERROR_DATA_TRUNCATED),
      |                                                                                                                           ^
/builddir/build/BUILD/blend2d-5a263ce51f3f880ee6c60f6345d18c3eccbe200f/src/blend2d/unicode/unicode_test.cpp:31:59: error: expected primary-expression before ‘{’ token
   31 |     #define ENTRY(DST, DST_ENC, SRC, SRC_ENC, ERROR_CODE) { \
      |                                                           ^
/builddir/build/BUILD/blend2d-5a263ce51f3f880ee6c60f6345d18c3eccbe200f/src/blend2d/unicode/unicode_test.cpp:99:5: note: in expansion of macro ‘ENTRY’
   99 |     ENTRY(""                                , UTF8  , "\xF4"                            , UTF8  , BL_ERROR_DATA_TRUNCATED),
      |     ^~~~~
/builddir/build/BUILD/blend2d-5a263ce51f3f880ee6c60f6345d18c3eccbe200f/src/blend2d/unicode/unicode_test.cpp:99:123: error: expected primary-expression before ‘,’ token
   99 |     ENTRY(""                                , UTF8  , "\xF4"                            , UTF8  , BL_ERROR_DATA_TRUNCATED),
      |                                                                                                                           ^
/builddir/build/BUILD/blend2d-5a263ce51f3f880ee6c60f6345d18c3eccbe200f/src/blend2d/unicode/unicode_test.cpp:31:59: error: expected primary-expression before ‘{’ token
   31 |     #define ENTRY(DST, DST_ENC, SRC, SRC_ENC, ERROR_CODE) { \
      |                                                           ^
/builddir/build/BUILD/blend2d-5a263ce51f3f880ee6c60f6345d18c3eccbe200f/src/blend2d/unicode/unicode_test.cpp:100:5: note: in expansion of macro ‘ENTRY’
  100 |     ENTRY(""                                , UTF8  , "\xF4\x8F"                        , UTF8  , BL_ERROR_DATA_TRUNCATED),
      |     ^~~~~
/builddir/build/BUILD/blend2d-5a263ce51f3f880ee6c60f6345d18c3eccbe200f/src/blend2d/unicode/unicode_test.cpp:100:123: error: expected primary-expression before ‘,’ token
  100 |     ENTRY(""                                , UTF8  , "\xF4\x8F"                        , UTF8  , BL_ERROR_DATA_TRUNCATED),
      |                                                                                                                           ^
/builddir/build/BUILD/blend2d-5a263ce51f3f880ee6c60f6345d18c3eccbe200f/src/blend2d/unicode/unicode_test.cpp:31:59: error: expected primary-expression before ‘{’ token
   31 |     #define ENTRY(DST, DST_ENC, SRC, SRC_ENC, ERROR_CODE) { \
      |                                                           ^
/builddir/build/BUILD/blend2d-5a263ce51f3f880ee6c60f6345d18c3eccbe200f/src/blend2d/unicode/unicode_test.cpp:101:5: note: in expansion of macro ‘ENTRY’
  101 |     ENTRY(""                                , UTF8  , "\xF4\x8F\xBF"                    , UTF8  , BL_ERROR_DATA_TRUNCATED),
      |     ^~~~~
/builddir/build/BUILD/blend2d-5a263ce51f3f880ee6c60f6345d18c3eccbe200f/src/blend2d/unicode/unicode_test.cpp:101:123: error: expected primary-expression before ‘,’ token
  101 |     ENTRY(""                                , UTF8  , "\xF4\x8F\xBF"                    , UTF8  , BL_ERROR_DATA_TRUNCATED),
      |                                                                                                                           ^
/builddir/build/BUILD/blend2d-5a263ce51f3f880ee6c60f6345d18c3eccbe200f/src/blend2d/unicode/unicode_test.cpp:31:59: error: expected primary-expression before ‘{’ token
   31 |     #define ENTRY(DST, DST_ENC, SRC, SRC_ENC, ERROR_CODE) { \
      |                                                           ^
/builddir/build/BUILD/blend2d-5a263ce51f3f880ee6c60f6345d18c3eccbe200f/src/blend2d/unicode/unicode_test.cpp:104:5: note: in expansion of macro ‘ENTRY’
  104 |     ENTRY("a"                               , UTF8  , "a\xF4\x8F\xBF"                   , UTF8  , BL_ERROR_DATA_TRUNCATED),
      |     ^~~~~
/builddir/build/BUILD/blend2d-5a263ce51f3f880ee6c60f6345d18c3eccbe200f/src/blend2d/unicode/unicode_test.cpp:104:123: error: expected primary-expression before ‘,’ token
  104 |     ENTRY("a"                               , UTF8  , "a\xF4\x8F\xBF"                   , UTF8  , BL_ERROR_DATA_TRUNCATED),
      |                                                                                                                           ^
/builddir/build/BUILD/blend2d-5a263ce51f3f880ee6c60f6345d18c3eccbe200f/src/blend2d/unicode/unicode_test.cpp:31:59: error: expected primary-expression before ‘{’ token
   31 |     #define ENTRY(DST, DST_ENC, SRC, SRC_ENC, ERROR_CODE) { \
      |                                                           ^
/builddir/build/BUILD/blend2d-5a263ce51f3f880ee6c60f6345d18c3eccbe200f/src/blend2d/unicode/unicode_test.cpp:105:5: note: in expansion of macro ‘ENTRY’
  105 |     ENTRY("ab"                              , UTF8  , "ab\xF4\x8F\xBF"                  , UTF8  , BL_ERROR_DATA_TRUNCATED),
      |     ^~~~~
/builddir/build/BUILD/blend2d-5a263ce51f3f880ee6c60f6345d18c3eccbe200f/src/blend2d/unicode/unicode_test.cpp:105:123: error: expected primary-expression before ‘,’ token
  105 |     ENTRY("ab"                              , UTF8  , "ab\xF4\x8F\xBF"                  , UTF8  , BL_ERROR_DATA_TRUNCATED),
      |                                                                                                                           ^
/builddir/build/BUILD/blend2d-5a263ce51f3f880ee6c60f6345d18c3eccbe200f/src/blend2d/unicode/unicode_test.cpp:31:59: error: expected primary-expression before ‘{’ token
   31 |     #define ENTRY(DST, DST_ENC, SRC, SRC_ENC, ERROR_CODE) { \
      |                                                           ^
/builddir/build/BUILD/blend2d-5a263ce51f3f880ee6c60f6345d18c3eccbe200f/src/blend2d/unicode/unicode_test.cpp:106:5: note: in expansion of macro ‘ENTRY’
  106 |     ENTRY("TestString"                      , UTF8  , "TestString\xC5"                  , UTF8  , BL_ERROR_DATA_TRUNCATED),
      |     ^~~~~
/builddir/build/BUILD/blend2d-5a263ce51f3f880ee6c60f6345d18c3eccbe200f/src/blend2d/unicode/unicode_test.cpp:106:123: error: expected primary-expression before ‘,’ token
  106 |     ENTRY("TestString"                      , UTF8  , "TestString\xC5"                  , UTF8  , BL_ERROR_DATA_TRUNCATED),
      |                                                                                                                           ^
/builddir/build/BUILD/blend2d-5a263ce51f3f880ee6c60f6345d18c3eccbe200f/src/blend2d/unicode/unicode_test.cpp:31:59: error: expected primary-expression before ‘{’ token
   31 |     #define ENTRY(DST, DST_ENC, SRC, SRC_ENC, ERROR_CODE) { \
      |                                                           ^
/builddir/build/BUILD/blend2d-5a263ce51f3f880ee6c60f6345d18c3eccbe200f/src/blend2d/unicode/unicode_test.cpp:110:5: note: in expansion of macro ‘ENTRY’
  110 |     ENTRY("\0T\0e\0s\0t\0S\0t\0r\0i\0n\0g"  , UTF16 , "TestString\xC5"                  , UTF8  , BL_ERROR_DATA_TRUNCATED),
      |     ^~~~~
/builddir/build/BUILD/blend2d-5a263ce51f3f880ee6c60f6345d18c3eccbe200f/src/blend2d/unicode/unicode_test.cpp:110:123: error: expected primary-expression before ‘,’ token
  110 |     ENTRY("\0T\0e\0s\0t\0S\0t\0r\0i\0n\0g"  , UTF16 , "TestString\xC5"                  , UTF8  , BL_ERROR_DATA_TRUNCATED),
      |                                                                                                                           ^
/builddir/build/BUILD/blend2d-5a263ce51f3f880ee6c60f6345d18c3eccbe200f/src/blend2d/unicode/unicode_test.cpp:31:59: error: expected primary-expression before ‘{’ token
   31 |     #define ENTRY(DST, DST_ENC, SRC, SRC_ENC, ERROR_CODE) { \
      |                                                           ^
/builddir/build/BUILD/blend2d-5a263ce51f3f880ee6c60f6345d18c3eccbe200f/src/blend2d/unicode/unicode_test.cpp:114:5: note: in expansion of macro ‘ENTRY’
  114 |     ENTRY(""                                , UTF8  , "\x80"                            , UTF8  , BL_ERROR_INVALID_STRING),
      |     ^~~~~
/builddir/build/BUILD/blend2d-5a263ce51f3f880ee6c60f6345d18c3eccbe200f/src/blend2d/unicode/unicode_test.cpp:114:123: error: expected primary-expression before ‘,’ token
  114 |     ENTRY(""                                , UTF8  , "\x80"                            , UTF8  , BL_ERROR_INVALID_STRING),
      |                                                                                                                           ^
/builddir/build/BUILD/blend2d-5a263ce51f3f880ee6c60f6345d18c3eccbe200f/src/blend2d/unicode/unicode_test.cpp:31:59: error: expected primary-expression before ‘{’ token
   31 |     #define ENTRY(DST, DST_ENC, SRC, SRC_ENC, ERROR_CODE) { \
      |                                                           ^
/builddir/build/BUILD/blend2d-5a263ce51f3f880ee6c60f6345d18c3eccbe200f/src/blend2d/unicode/unicode_test.cpp:115:5: note: in expansion of macro ‘ENTRY’
  115 |     ENTRY(""                                , UTF8  , "\xC1"                            , UTF8  , BL_ERROR_INVALID_STRING),
      |     ^~~~~
/builddir/build/BUILD/blend2d-5a263ce51f3f880ee6c60f6345d18c3eccbe200f/src/blend2d/unicode/unicode_test.cpp:115:123: error: expected primary-expression before ‘,’ token
  115 |     ENTRY(""                                , UTF8  , "\xC1"                            , UTF8  , BL_ERROR_INVALID_STRING),
      |                                                                                                                           ^
/builddir/build/BUILD/blend2d-5a263ce51f3f880ee6c60f6345d18c3eccbe200f/src/blend2d/unicode/unicode_test.cpp:31:59: error: expected primary-expression before ‘{’ token
   31 |     #define ENTRY(DST, DST_ENC, SRC, SRC_ENC, ERROR_CODE) { \
      |                                                           ^
/builddir/build/BUILD/blend2d-5a263ce51f3f880ee6c60f6345d18c3eccbe200f/src/blend2d/unicode/unicode_test.cpp:116:5: note: in expansion of macro ‘ENTRY’
  116 |     ENTRY(""                                , UTF8  , "\xF5\x8F\xBF\xBF"                , UTF8  , BL_ERROR_INVALID_STRING),
      |     ^~~~~
/builddir/build/BUILD/blend2d-5a263ce51f3f880ee6c60f6345d18c3eccbe200f/src/blend2d/unicode/unicode_test.cpp:116:123: error: expected primary-expression before ‘,’ token
  116 |     ENTRY(""                                , UTF8  , "\xF5\x8F\xBF\xBF"                , UTF8  , BL_ERROR_INVALID_STRING),
      |                                                                                                                           ^
/builddir/build/BUILD/blend2d-5a263ce51f3f880ee6c60f6345d18c3eccbe200f/src/blend2d/unicode/unicode_test.cpp:31:59: error: expected primary-expression before ‘{’ token
   31 |     #define ENTRY(DST, DST_ENC, SRC, SRC_ENC, ERROR_CODE) { \
      |                                                           ^
/builddir/build/BUILD/blend2d-5a263ce51f3f880ee6c60f6345d18c3eccbe200f/src/blend2d/unicode/unicode_test.cpp:117:5: note: in expansion of macro ‘ENTRY’
  117 |     ENTRY(""                                , UTF8  , "\x91\x8F\xBF\xBF"                , UTF8  , BL_ERROR_INVALID_STRING),
      |     ^~~~~
/builddir/build/BUILD/blend2d-5a263ce51f3f880ee6c60f6345d18c3eccbe200f/src/blend2d/unicode/unicode_test.cpp:117:123: error: expected primary-expression before ‘,’ token
  117 |     ENTRY(""                                , UTF8  , "\x91\x8F\xBF\xBF"                , UTF8  , BL_ERROR_INVALID_STRING),
      |                                                                                                                           ^
/builddir/build/BUILD/blend2d-5a263ce51f3f880ee6c60f6345d18c3eccbe200f/src/blend2d/unicode/unicode_test.cpp:31:59: error: expected primary-expression before ‘{’ token
   31 |     #define ENTRY(DST, DST_ENC, SRC, SRC_ENC, ERROR_CODE) { \
      |                                                           ^
/builddir/build/BUILD/blend2d-5a263ce51f3f880ee6c60f6345d18c3eccbe200f/src/blend2d/unicode/unicode_test.cpp:118:5: note: in expansion of macro ‘ENTRY’
  118 |     ENTRY(""                                , UTF8  , "\xF6\x8F\xBF\xBF"                , UTF8  , BL_ERROR_INVALID_STRING),
      |     ^~~~~
/builddir/build/BUILD/blend2d-5a263ce51f3f880ee6c60f6345d18c3eccbe200f/src/blend2d/unicode/unicode_test.cpp:118:123: error: expected primary-expression before ‘,’ token
  118 |     ENTRY(""                                , UTF8  , "\xF6\x8F\xBF\xBF"                , UTF8  , BL_ERROR_INVALID_STRING),
      |                                                                                                                           ^
/builddir/build/BUILD/blend2d-5a263ce51f3f880ee6c60f6345d18c3eccbe200f/src/blend2d/unicode/unicode_test.cpp:31:59: error: expected primary-expression before ‘{’ token
   31 |     #define ENTRY(DST, DST_ENC, SRC, SRC_ENC, ERROR_CODE) { \
      |                                                           ^
/builddir/build/BUILD/blend2d-5a263ce51f3f880ee6c60f6345d18c3eccbe200f/src/blend2d/unicode/unicode_test.cpp:119:5: note: in expansion of macro ‘ENTRY’
  119 |     ENTRY(""                                , UTF8  , "\xF4\xFF\xBF\xBF"                , UTF8  , BL_ERROR_INVALID_STRING),
      |     ^~~~~
/builddir/build/BUILD/blend2d-5a263ce51f3f880ee6c60f6345d18c3eccbe200f/src/blend2d/unicode/unicode_test.cpp:119:123: error: expected primary-expression before ‘,’ token
  119 |     ENTRY(""                                , UTF8  , "\xF4\xFF\xBF\xBF"                , UTF8  , BL_ERROR_INVALID_STRING),
      |                                                                                                                           ^
/builddir/build/BUILD/blend2d-5a263ce51f3f880ee6c60f6345d18c3eccbe200f/src/blend2d/unicode/unicode_test.cpp:31:59: error: expected primary-expression before ‘{’ token
   31 |     #define ENTRY(DST, DST_ENC, SRC, SRC_ENC, ERROR_CODE) { \
      |                                                           ^
/builddir/build/BUILD/blend2d-5a263ce51f3f880ee6c60f6345d18c3eccbe200f/src/blend2d/unicode/unicode_test.cpp:122:5: note: in expansion of macro ‘ENTRY’
  122 |     ENTRY(""                                , UTF8  , "\xC0\xA0"                        , UTF8  , BL_ERROR_INVALID_STRING)
      |     ^~~~~
/builddir/build/BUILD/blend2d-5a263ce51f3f880ee6c60f6345d18c3eccbe200f/src/blend2d/unicode/unicode_test.cpp:30:26: warning: unused variable ‘testEntries’ [-Wunused-variable]
   30 |   static const TestEntry testEntries[] = {
      |                          ^~~~~~~~~~~
/builddir/build/BUILD/blend2d-5a263ce51f3f880ee6c60f6345d18c3eccbe200f/src/blend2d/unicode/unicode_test.cpp: At global scope:
/builddir/build/BUILD/blend2d-5a263ce51f3f880ee6c60f6345d18c3eccbe200f/src/blend2d/unicode/unicode_test.cpp:127:3: error: expected unqualified-id before ‘for’
  127 |   for (size_t i = 0; i < BL_ARRAY_SIZE(testEntries); i++) {
      |   ^~~
/builddir/build/BUILD/blend2d-5a263ce51f3f880ee6c60f6345d18c3eccbe200f/src/blend2d/unicode/unicode_test.cpp:127:22: error: ‘i’ does not name a type
  127 |   for (size_t i = 0; i < BL_ARRAY_SIZE(testEntries); i++) {
      |                      ^
/builddir/build/BUILD/blend2d-5a263ce51f3f880ee6c60f6345d18c3eccbe200f/src/blend2d/unicode/unicode_test.cpp:127:54: error: ‘i’ does not name a type
  127 |   for (size_t i = 0; i < BL_ARRAY_SIZE(testEntries); i++) {
      |                                                      ^
/builddir/build/BUILD/blend2d-5a263ce51f3f880ee6c60f6345d18c3eccbe200f/src/blend2d/unicode/unicode_test.cpp:347:1: error: expected declaration before ‘}’ token
  347 | } // {bl}
      | ^
[301/329] /usr/bin/g++  -I/builddir/build/BUILD/blend2d-5a263ce51f3f880ee6c60f6345d18c3eccbe200f/src -O2 -flto=auto -ffat-lto-objects -fexceptions -g -grecord-gcc-switches -pipe -Wall -Werror=format-security -Wp,-U_FORTIFY_SOURCE,-D_FORTIFY_SOURCE=3 -Wp,-D_GLIBCXX_ASSERTIONS -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -fstack-protector-strong -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1  -m64 -march=z13 -mtune=z14 -fasynchronous-unwind-tables -fstack-clash-protection -O2 -g -DNDEBUG -fvisibility=hidden -Wall -Wextra -fno-exceptions -fno-rtti -fno-math-errno -fno-trapping-math -fno-finite-math-only -fno-threadsafe-statics -fno-semantic-interposition -DBL_BUILD_NO_JIT -DBL_TEST -DBL_STATIC -O2 -fno-enforce-eh-specs -fmerge-all-constants -ftree-vectorize -MD -MT CMakeFiles/bl_test_unit.dir/src/blend2d/unicode/unicode.cpp.o -MF CMakeFiles/bl_test_unit.dir/src/blend2d/unicode/unicode.cpp.o.d -o CMakeFiles/bl_test_unit.dir/src/blend2d/unicode/unicode.cpp.o -c /builddir/build/BUILD/blend2d-5a263ce51f3f880ee6c60f6345d18c3eccbe200f/src/blend2d/unicode/unicode.cpp
[302/329] /usr/bin/g++  -I/builddir/build/BUILD/blend2d-5a263ce51f3f880ee6c60f6345d18c3eccbe200f/src -O2 -flto=auto -ffat-lto-objects -fexceptions -g -grecord-gcc-switches -pipe -Wall -Werror=format-security -Wp,-U_FORTIFY_SOURCE,-D_FORTIFY_SOURCE=3 -Wp,-D_GLIBCXX_ASSERTIONS -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -fstack-protector-strong -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1  -m64 -march=z13 -mtune=z14 -fasynchronous-unwind-tables -fstack-clash-protection -O2 -g -DNDEBUG -fvisibility=hidden -Wall -Wextra -fno-exceptions -fno-rtti -fno-math-errno -fno-trapping-math -fno-finite-math-only -fno-threadsafe-statics -fno-semantic-interposition -DBL_BUILD_NO_JIT -DBL_TEST -DBL_STATIC -O2 -fno-enforce-eh-specs -fmerge-all-constants -ftree-vectorize -MD -MT CMakeFiles/bl_test_unit.dir/src/blend2d/raster/rastercontextops.cpp.o -MF CMakeFiles/bl_test_unit.dir/src/blend2d/raster/rastercontextops.cpp.o.d -o CMakeFiles/bl_test_unit.dir/src/blend2d/raster/rastercontextops.cpp.o -c /builddir/build/BUILD/blend2d-5a263ce51f3f880ee6c60f6345d18c3eccbe200f/src/blend2d/raster/rastercontextops.cpp
ninja: build stopped: subcommand failed.
```

</details>

detailed build log: https://kojipkgs.fedoraproject.org//work/tasks/2281/114792281/build.log

--- a/src/blend2d/unicode/unicode_test.cpp
+++ b/src/blend2d/unicode/unicode_test.cpp
@@ -87,7 +87,7 @@ UNIT(unicode, BL_TEST_GROUP_CORE_UTILITIES) {
     ENTRY("Test"                            , UTF8  , "\0\0\0T\0\0\0e\0\0\0s\0\0\0t"    , UTF32 , BL_SUCCESS),
     ENTRY("\0T\0e\0s\0t"                    , UTF16 , "\0\0\0T\0\0\0e\0\0\0s\0\0\0t"    , UTF32 , BL_SUCCESS),
     ENTRY("\0\0\0T\0\0\0e\0\0\0s\0\0\0t"    , UTF32 , "\0\0\0T\0\0\0e\0\0\0s\0\0\0t"    , UTF32 , BL_SUCCESS),
-    ENTRY("\0\0\0T\0\0\0e\0\0\0s\0\0\0t"    , UTF32 , "\0T\0e\0s\0t"                    , UTF16 , BL_SUCCESS)
+    ENTRY("\0\0\0T\0\0\0e\0\0\0s\0\0\0t"    , UTF32 , "\0T\0e\0s\0t"                    , UTF16 , BL_SUCCESS),
     ENTRY("\0\0\0T\0\0\0e\0\0\0s\0\0\0t"    , UTF32 , "Test"                            , LATIN1, BL_SUCCESS),
     ENTRY("\0\0\0T\0\0\0e\0\0\0s\0\0\0t"    , UTF32 , "Test"                            , UTF8  , BL_SUCCESS),
     #endif

